apps/aecore/test/aec_test_utils.erl

```diff
@@ -359,18 +359,24 @@ next_block_with_state([{PB, PBS} | _] = Chain, Target, Time0, TxsFun, Nonce,
                       PubKey, PrivKey, BeneficiaryPubKey) ->
     Height = aec_blocks:height(PB) + 1,
     Txs = TxsFun(Height),
-    %% NG: if a block X used to have Txs, now put them in a microblock just before
-    %% the key-block at height X.
+    %% NG: if a block X used to have Txs, now put them in micro-blocks just before
+    %% the key-block at height X. Every transaction is put in separate micro-block.
```

Comment:
I see this constrain of one tx per microblock is a simplification of the test helper as it is what needed so far in test cases. Ok.